index.html

```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <title>lesson-2</title>
+  </head>
+  <body>
+    <h1>Моя первая страница</h1>
+    <p><a href="/about">Об авторе</a></p>
```

Comment:
Check if the link works in chrome properly.

Reply:
this link also works but i changed it